airbyte-integrations/connectors/source-dynamodb/src/main/java/io/airbyte/integrations/source/dynamodb/DynamodbSource.java

```diff
@@ -146,14 +146,15 @@ private AutoCloseableIterator<AirbyteMessage> scanIncremental(final DynamodbOper
 
     // cursor type will be retrieved from the json schema to save time on db schema crawling reading
     // large amount of items
-    final String cursorType = properties.get(cursorField).get("type").asText();
+    final String cursorType = properties.get(cursorField).get("type").toString();
+    LOGGER.info("cursor type: {}", cursorType);
```

Comment:
Does the rest of the source use this logging pattern, or a different one?

Reply:
as far as i see - yes